contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -426,11 +426,12 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
 
         /**
          * When rebalance is called for the first time there is no strategy
-         * liquidity in the pool yet. The full liquidity removal is thus skipped.
+         * liquidity in the pool yet. The liquidity removal is thus skipped.
          */
         if (tokenId != 0) {
-            _removeLiquidity(1e18);
+            _removeLiquidityToEnsureSwap(_amountToSwap, _swapWeth);
```

Comment:
nit: Can probably replace lines 431 to 439 with something like this. But not a big deal though since `_removeLiquidityToEnsureSwap` has a `amountToSwap > 0` check 
```
        if (_amountToSwap > 0) {
            if (tokenId != 0) {
                _removeLiquidityToEnsureSwap(_amountToSwap, _swapWeth);
            }

            _swapToDesiredPosition(_amountToSwap, _swapWeth, _minTokenReceived);
        }
        ```

Reply:
Yes.  I think this would be cleaner.